doc/index.rst

```diff
@@ -0,0 +1,72 @@
+ur_description
+==============
+
+This package contains the kinematic and visual description of Universal Robots manipulators for ROS
+2.
+
+Structure of the repository
+---------------------------
+
+As the kinematics of all models is roughly the same and varies only in the meshes, the DH
+parameters and the physical parameters the description uses xacro macros in order to make it reuse
+as much as possible. This way it is less likely that errors are introduced in one of the models.
```

Comment:
```suggestion
As the kinematics of all models is roughly the same and varies only in the meshes, the DH
parameters and the physical parameters; the description uses xacro macros in order to make it reuse
as much as possible. This way it is less likely that errors are introduced in one of the models.
```
or 

```suggestion
The description uses xacro macros in order to maximize reuse, since the kinematics is roughly the same for all models, varying only in the meshes, the DH
parameters and the physical parameters. This way it is less likely that errors are introduced in one of the models.
```